packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -181,10 +181,15 @@ const createStaticPageListItems = (localDataManifest: LocalDataManifest) => {
   );
 };
 
-const yextrcContents: string = fs.readFileSync(".yextrc", "utf8");
-const parsedContents = YAML.parse(yextrcContents);
-const accountId: string = parsedContents.accountId;
-const universe: string = parsedContents.universe;
+let accountId = "";
+let universe = "false";
```

Comment:
Don't set this to an arbitrary value. Same for accountId. Just let them be undefined.

Reply:
Oh! Thank you for the reminder.